desc/compat.py

```diff
@@ -261,3 +261,66 @@ def rescale(
     eq.surface = eq.get_surface_at(rho=1)
 
     return eq
+
+
+def rotate_zeta(eq, angle, copy=False):
+    """Rotate the equilibrium about the toroidal direction.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium to rotate.
+    angle : float
+        Angle to rotate the equilibrium in radians. The actual physical rotation
+        is by angle / self.NFP.
```

Comment:
i'd prefer if angle is the actual rotation, and the user should include nfp explicitly

Reply:
@dpanici also wanted this. I will change it.